api/battle.php

```diff
@@ -13,80 +13,159 @@
 }
 # End standard auth
 
-
-$battle_result = $system->db->query("SELECT battle_type FROM battles WHERE `battle_id`='{$player->battle_id}' LIMIT 1");
-if($system->db->last_num_rows) {
-    $battle_data = $system->db->fetch($battle_result);
-
-    $battle_route = null;
-    foreach(Router::$routes as $page_id => $page) {
-        if(empty($page->battle_type)) {
-            continue;
+if (isset($_POST['request'])) {
```

Comment:
Hmm, could we actually put this in the Travel API? I know technically this is a "battle" related action, but as is this API is pretty cleanly defined and only handles in-progress battles. The Travel API has more of an assorted collection of travel-related functionality related to movement and nearby players, where I think this would fit in as one other travel-related action

Reply:
Moved to travel.